.github/CHANGELOG.md

```diff
@@ -20,7 +21,7 @@
 
 This release contains contributions from (in alphabetical order):
 
-[Joost Bus](https://github.com/soosub), [Sebastián Duque Mesa](https://github.com/sduquemesa), [Luis Mantilla](https://github.com/BestQuark)
+[Joost Bus](https://github.com/soosub), [Sebastián Duque Mesa](https://github.com/sduquemesa), [Luis Mantilla](https://github.com/BestQuark), Nariman Saadatmand
```

Comment:
Don't forget to add your github handle 📲
```suggestion
[Joost Bus](https://github.com/soosub), [Sebastián Duque Mesa](https://github.com/sduquemesa), [Luis Mantilla](https://github.com/BestQuark), [Nariman Saadatmand](https://github.com/nariman87)
```

Reply:
Let's avoid doing that temporarily, for now, thanks.